cli/azd/pkg/auth/manager.go

```diff
@@ -638,13 +664,21 @@ func (m *Manager) Logout(ctx context.Context) error {
 
 	// we are fine to ignore the error here, it just means there's nothing to clean up.
 	currentUser, _ := readUserProperties(cfg)
-
-	// When logged in as a service principal, remove the stored credential
-	if currentUser != nil && currentUser.TenantID != nil && currentUser.ClientID != nil {
-		if err := m.saveLoginForServicePrincipal(
-			*currentUser.TenantID, *currentUser.ClientID, &persistedSecret{},
-		); err != nil {
-			return fmt.Errorf("removing authentication secrets: %w", err)
+	if currentUser != nil {
+		if currentUser.FromOneAuth {
+			// TODO: read GlobalCommandOptions.EnableDebugLogging
```

Comment:
The `Debug` options determined only whether to enable OneAuth logging. They aren't necessary anymore because I have OneAuth calling `log.Print`, which allows `main()` to decide whether the spew goes to the console.